src/game/UICmdEdit.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+#include "GameDef.h"
+#include "N3Base/N3UIBase.h"
+#include "N3Base/N3UIString.h"
+#include "N3Base/N3UIEdit.h"
+
+//////////////////////////////////////////////////////////////////////
+
+class CUICmdEditDlg : public CN3UIBase {
```

Comment:
Officially the class name is simply named `CUICmdEdit`, I assume based on the leftover comments in the source file, you originally created it with the 'Dlg' suffix and then renamed it without renaming the class itself.

I don't know if this project intends to keep official class names for things like this, but if so, this should be renamed `CUICmdEdit`.

Reply:
Ideally we keep the same official names. 100% with you @twostars.